### ISSUE TITLE
LineageOS 14.1 branch

### ISSUE THREAD

**Issue by bocki** [Meticulus/android_device_huawei_hi6250#5]
Will you be releasing the current set of patches needed to compile your LineageOS 14.1 builds?

**Comment by Meticulus** [comment]
Maybe... One day... :+1: 

Why, you got something to contribute?

**Comment by bocki** [comment]
My own attempts to get the current source to boot failed, even though I succeeded in compiling, so I guess changes are not (exclusively) trivial. Any pointers would be appreciated.

It's hard to contribute without any available source code. I wrote patches for some of my previous devices, and I think it would help immensely to have access to a working build system. Reinventing the wheel can be time-consuming...

**Comment by Meticulus** [comment]
For now, the source will remain closed. I worked on the "wheel" very hard and no one appreciated the "wheel" which, in itself is no big deal, but then I started getting harassed by n00bs, so now my "wheel" is closed source.

This of course is not your fault and I'm sorry that things are the way they are.

I will say this, getting nougat to boot, starting with the Lineage 13 source is decidedly difficult. The biggest problem is SELinux. I had to disable it, which is "not a supported configuration in Nougat".

Good Luck... :1st_place_medal:
